File: BD/Classe/Heure.php
<?php
class Heure{
    private $_ID;
    private $_hDeb;
    private $_hFin;

    public function __construct($id,$deb,$fin)
    {
        $this->_ID = $id;
        $this->_hDeb = $deb;
        $this->_hFin = $fin;
    }

    public function getID(){
        return $this->_ID;
    }

    public function getDeb(){
        return $this->_hDeb;
    }

    public function getFin(){
        return $this->_hFin;
    }

}
?>

File: MyApplication/app/src/main/java/com/example/olivet/myapplication/Heure.java
package com.example.olivet.myapplication;

/**
 * Created by schultz on 16/01/18.
 */

public class Heure {

    private int idHeure;
    private String hDeb;
    private String hFin;

    // Constructeur
    public Heure(int id, String deb, String fin) {
        this.idHeure=id;
        this.hDeb=deb;
        this.hFin=fin;
    }

    public int getIdHeure() {
        return idHeure;
    }

    public void setIdHeure(int idHeure) {
        this.idHeure = idHeure;
    }

    public String gethDeb() {
        return hDeb;
    }

    public void sethDeb(String hDeb) {
        this.hDeb = hDeb;
    }

    public String gethFin() {
        return hFin;
    }

    public void sethFin(String hFin) {
        this.hFin = hFin;
    }
}
